README.md

```diff
@@ -20,6 +20,7 @@ Please report issues at the [go-flutter issue tracker](https://github.com/go-flu
 - [file_picker](https://github.com/miguelpruivo/flutter_file_picker) - Select single or multiple file paths using the native file explorer. ([pub.dev](https://pub.dev/packages/file_picker))
 - [sqlite](https://github.com/nealwon/go-flutter-plugin-sqlite) - Flutter plugin for SQLite. ([pub.dev](https://pub.dev/packages/sqflite))
 - [platform_device_id](https://github.com/BestBurning/platform_device_id) - Query device identifier. ([pub.dev](https://pub.dev/packages/platform_device_id))
+- [shutdown_platform](https://github.com/BestBurning/shutdown_platform) - Shutdown the machine. ([pub.dev](https://pub.dev/packages/shutdown_platform)) **Only for windows/linux/osx**
```

Comment:
windows/linux/osx are the only go-flutter supported platforms.
You can remove '**Only for windows/linux/osx**'

Reply:
.....the plugin it's  **not support  android | ios** now

Reply:
ok